Backend_Mapanimal/Controllers/RaceController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Backend_Mapanimal.Entities;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+
+namespace Backend_Mapanimal.Controllers
+{
+
+    [Route("api/race")]
+    [ApiController]
+    public class RaceController : ControllerBase
+    {
+        private readonly ILogger<RaceController> logger;
+
+        public RaceController(ILogger<RaceController> logger)
+        {
+            this.logger = logger;
+        }
+
+        [HttpGet]
+        public ActionResult <List<Race>> Get()// Get general
```

Comment:
Please use async Tasks as response type for all the endpoints, we don't want to block the main thread of the API process